src/simulator_bridge.cpp

```diff
@@ -81,6 +92,90 @@ void modelStatesCallback(const gazebo_msgs::ModelStates& msg)
             depth_msg.header.stamp = ros::Time::now();
             depth_pub.publish(depth_msg);
         }
+
+        /*
+         * If the pinger index isn't found, break out and dont perform
+         * hydrophone calculations.
+         */
+        if (pinger_index > -1)
```

Comment:
might be a good idea to add an error or warning message if we can't find the pinger index.